examples/resources/eva_role.tf

```diff
@@ -0,0 +1,6 @@
+
+resource "eva_role" "example" {
+  name          = "My Example Role"
+  user_type     = 1
```

Comment:
How do we keep this list updated since we have no control under it ? I'd say for us to leave all validations in EVA API and we just proxy these values to them. what do you think ?